spec/features/show_page_spec.rb

```diff
@@ -1,21 +1,20 @@
 require "rails_helper"
 
 RSpec.describe "customer show page" do
-
   it "displays the customers orders paginated" do
     customer = create(:customer)
-    first_order, _a, _b, last_order = Array.new(4) { create(:order, customer: customer) }
+    orders = create_list(:order, 4, customer: customer)
 
     visit admin_customer_path(customer)
     within('.attribute-data--has-many') do
-      expect(page).to have_content(first_order.id)
-      expect(page).to have_no_content(last_order.id)
+      expect(page).to have_order_row(orders.first.id)
+      expect(page).not_to have_order_row(orders.last.id)
     end
-
     click_on("Next ›")
+
     within('.attribute-data--has-many') do
-      expect(page).to have_content(last_order.id)
-      expect(page).to have_no_content(first_order.id)
+      expect(page).to have_order_row(orders.last.id)
+      expect(page).not_to have_order_row(orders.first.id)
     end
   end
```

Comment:
@jayroh I tried to address this in 48b0e2f38e669d5230f3480f5c1600bbf0566395, let me know what you think!